default/python/AI/empire/README.md

```diff
@@ -0,0 +1,47 @@
+# Empire info module
+
+This module contains code that responsible for holding useful information about empire objects.
+
+## Implementation details. 
+
+Files in this module wildly use encapsulation principle.
+
+Data container itself is hidden from outer scope and access to it should be provided via functions.
+This approach allows changing internal implementation and add new functionality 
+without changing code which uses this module. 
+Also, we could keуp this module as simple as required for current code.
+
+Modules usually don't have any AI dependencies, so they could be easily used from any part of AI code.
+
+Each module have fucntion to set data and functions to get data. This gives us a temporal coupling. 
```

Comment:
`have fucntion` -> `has functions` or `has a function` (not sure how many functions are present)